src/recordlinker/models.py

```diff
@@ -0,0 +1,40 @@
+import uuid
```

Comment:
Small nitpick on the location of this module.

The DIBBs team originally create two packages for this building block container; `app` and `app.linkage`.  From what I gather, the intent was to put the code specific to the algorithm itself and how it functions inside `app.linkage`, where `app` was intended to store code related to the API and processing the data received and sent.  Now there is one glaring omission to this strategy in that all the FHIR parsing code was also stuffed into the `app.linkage` package, as the algorithm really doesn't **need** FHIR to function, that aside though they did a decent job organizing the functionality.

I'd argue that `models.py` is key to how the algorithm works, as it describes the schema necessary for the blocking queries to work.  So for this I'd recommend we move this file to `recordlinker/linkage/models.py`.